src/test/java/wolox/training/UserRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,72 @@
+package wolox.training;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import wolox.training.models.Book;
+import wolox.training.models.Users;
+import wolox.training.repositories.BookRepository;
+import wolox.training.repositories.UsersRepository;
+
+import java.time.LocalDate;
+
+import static junit.framework.TestCase.assertTrue;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class UserRepositoryIntegrationTest {
+
+    @Autowired
+    private UsersRepository usersRepository;
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    private Users oneTestUser;
+    private Book oneTestBook;
+
+    @Before
+    public void SetUp() {
+        oneTestBook = new Book(
+                "1234",
+                "Song of ice and fire",
+                "G.R.Martin",
+                "Fantasy",
+                "A Game of Thrones",
+                "inset image",
+                "Bantam books",
+                1998
+        );
+        oneTestUser = new Users();
+        oneTestUser.setUsername("ocolmenares");
+        oneTestUser.setName("Oriana");
+        oneTestUser.setBirthday(LocalDate.parse("1997-11-03"));
+        oneTestUser.addBook(oneTestBook);
```

Comment:
Debería haber un constructor para un nuevo usuario así lo instanciamos como a los books, algo que te permita hacer esto:
```
oneTestUser = new User(
   "mede",
   "Nicolás Medero C.",
   ...
   ...
);
```
no? Qué opinamos?

Reply:
yes it sounds good, I've tried before but have some issues with LocalDate type, so that is why i leave it like that.